### ISSUE TITLE
Fix aria labels for footer

### ISSUE THREAD

**Issue by thedanfernandez** [NuevoFoundation/workshops#50]
Via Lighthouse audit - https://dequeuniversity.com/rules/axe/3.1/link-name?application=lighthouse 

Failing Elements
--
a.sc-gzVnrw.cZJBlR
a.sc-gzVnrw.cZJBlR
a.sc-gzVnrw.cZJBlR
a.sc-gzVnrw.cZJBlR
a.sc-fjdhpX.izNzAG.active
a.footer-pics
a.footer-pics
a.footer-pics
a.footer-pics
